pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.onsdigital.logging</groupId>
+    <groupId>com.github.onsdigital</groupId>
```

Comment:
NB. This change wasn't strictly necessary but all apps using this library import it (via JitPack) with this group-id. Making the one in the pom match meant that it was much easier to test locally by running `make install` and only changing the version to `1.0-SNAPSHOT` in the apps.